test/core/tsi/ssl_transport_security_utils_test.cc

```diff
@@ -429,6 +445,135 @@ INSTANTIATE_TEST_SUITE_P(FrameProtectorUtil, FlowTest,
 
 #endif  // OPENSSL_IS_BORINGSSL
 
+X509_CRL* read_crl(absl::string_view crl_string) {
```

Comment:
Can we move these "ReadCrl" and "ReadCert" methods to a test utility file so that we can reuse them, rather than duplicating the implementations?

Reply:
Yes - is there an existing test utils file that these fit in well?

Reply:
Done